homeassistant/components/camera/foscam.py

```diff
@@ -75,20 +75,32 @@ def motion_detection_enabled(self):
 
     def enable_motion_detection(self):
         """Enable motion detection in camera."""
-        ret, err = self._foscam_session.enable_motion_detection()
-        if ret == FOSCAM_COMM_ERROR:
-            _LOGGER.debug("Unable to communicate with Foscam Camera: %s", err)
-            self._motion_status = True
-        else:
+        try:
+            ret, err = self._foscam_session.enable_motion_detection()
+            if ret == FOSCAM_COMM_ERROR:
```

Comment:
This code makes no sense. If it is an error, log that an error has occurred and then assume the command was successful because we're setting the attribute to `True` ?

Reply:
If you loved this then you’ll love the constructor for this module. Who approved this in the first place???

Reply:
So if this is supposed to be the "positive" result, we should not log "Unable to communicate"? I know it is not your code, but if you're touching it, let's at least just remove that log statement.